vim/vscode-code-actions.vim

```diff
@@ -48,12 +48,14 @@ nnoremap <expr> == <SID>vscodeFormat() . '_'
 " gf/gF . Map to go to definition for now
 nnoremap <silent> gf :<C-u>call <SID>vscodeGoToDefinition()<CR>
 nnoremap <silent> gd :<C-u>call <SID>vscodeGoToDefinition()<CR>
+nnoremap <silent> gh :<C-u>call VSCodeNotify('editor.action.showHover')<CR>
 nnoremap <silent> <C-]> :<C-u>call <SID>vscodeGoToDefinition()<CR>
 nnoremap <silent> gF :<C-u>call VSCodeNotify('editor.action.peekDefinition')<CR>
 nnoremap <silent> gD :<C-u>call VSCodeNotify('editor.action.peekDefinition')<CR>
 
 xnoremap <silent> gf :<C-u>call <SID>vscodeGoToDefinition()<CR>
 xnoremap <silent> gd :<C-u>call <SID>vscodeGoToDefinition()<CR>
+xnoremap <silent> gh :<C-u>call VSCodeNotify('editor.action.showHover')<CR>
```

Comment:
This is not correct mapping - entering command line will exit visual mode which we probably don't want to do.
The correct one is to call `normal! gv` following by `VSCodeNotify()`, i.e.

```
function! s:hover()
  normal! gv
  call VSCodeNotify('editor.action.showHover)
endfunction

xnoremap <silent> gh :<C-u>call <SID>hover()<CR>
```